README.md

```diff
@@ -23,31 +23,65 @@ There are three modules this project:
       It instruments the java.util.logging.Logger.log using JavaAssist.
 
 4. riddles folder \
-    The riddles folder contains two riddles that can be solved using the profiler. 
+    The riddles folder contains two riddles that can be solved using the profiler you will write. 
     You can find there a `README` file with the run instructions. 
 
-Prepare work environment with Maven and IDE 
+Prepare work environment with GitHub Codespaces.
 --------------------------------------------
-To prepare your work environment, follow these steps:
+
+## 🧑‍💻 Option 1: Use VS Code in the Browser
+
+1. Go to the GitHub repository: [JavaInstrumentationWorkshop](https://github.com/irachle/JavaInstrumentationWorkshop)
+2. Click the green **Code** button.

```

Comment:
This is the only item in this list that ends with a `.` - let's be consistent and remove it?